content/layouts/page-react.html

```diff
@@ -4,8 +4,10 @@
   {% include "content/includes/breadcrumbs.html" %}
 {% endif %}
 
+{% if !loadingMessage %} {% assign loadingMessage = 'Please wait while we load the application for you.' %} {% endif %}
+
 <div id="content">
-<div class="main" role="main">
+<div id="main" class="main" role="main">
```

Comment:
> The `main` in `main` falls `main`ly on the `main`.

Am I doing it right?

Reply:
Haha we had that ID in most of the application pages so I figured I'd add it here to be safe.